k8s/aat/common/bsp/bulk-scan-processor.yaml

```
@@ -14,7 +14,7 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: bulk-scan-processor
-    version: 0.1.6
+    version: 0.1.8
```

0.1.9 is the latest version?
```suggestion
    version: 0.1.9
```